cmd/influxd/run.go

```diff
@@ -187,6 +188,38 @@ func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Ser
 			}
 		}
 
+		// Spin up any OpenTSDB servers
+		if config.OpenTSDB.Enabled {
+			o := config.OpenTSDB
+			db := opentsdb.DefaultDatabaseName
```

Comment:
We prefer to do this on the `config` object. In other words, the config object should have a method called "OpenTSDBDatabase()` (or something like that) and encapsulate the default-detection logic.